packages/renderer/src/get-extension-from-codec.ts

```diff
@@ -27,6 +27,8 @@ export const getFileExtensionFromCodec = (
 			return 'webm';
 		case 'vp9':
 			return 'webm';
+		case 'gif':
```

Comment:
On Lambda, the output works but looks horrible.
Also it only takes the first chunk - should we encode the chunks to another codec?

Also combineVideos() must then take `loop` and `skip-n-frames` into account

Reply:
Command to experiment with in `packages/example`:

pnpx remotion lambda render testbed framer --out-name=ggg.gif ggg.gif --codec=gif --log=verbose